plugins/woocommerce-admin/client/core-profiler/style.scss

```diff
@@ -344,3 +382,74 @@
 		margin-top: 20px;
 	}
 }
+
+.woocommerce-profiler-plugins {
+	display: flex;
+	flex-direction: column;
+	.woocommerce-profiler-plugins__content {
+		display: flex;
+		align-items: center;
+		align-self: center;
+		max-width: 1000px;
+		width: 100%;
+	}
+	.woocommerce-profiler-heading {
+		max-width: 615px;
+		margin: 58px 0 0 0;
+	}
+	.woocommerce-profiler-heading__subtitle {
+		margin: 0 0 48px 0 !important;
+		@include breakpoint( '<782px' ) {
+			margin-top: 12px !important;
+		}
+	}
+	.woocommerce-profiler-plugins__list {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+		gap: 16px;
+		flex: 50%;
+	}
+
+	.woocommerce-profiler-plugins-continue-button {
+		width: 200px;
+		height: 48px;
+		margin-top: 28px;
+		text-align: center;
+		display: block;
+	}
+
+	.plugin-error {
```

Comment:
It seems that the error message color is different. Also, plugin name needs to be bold.

```
/* Gutenberg/Gray 900 */

color: #1E1E1E;
```

![Screenshot 2023-05-24 at 11 39 11](https://github.com/woocommerce/woocommerce/assets/4344253/885534d4-b9f3-42e8-8fbc-99ce990b980f)

Expected:
![Screenshot 2023-05-24 at 11 39 46](https://github.com/woocommerce/woocommerce/assets/4344253/53910354-d19e-4b4d-8af8-9ad2ac064a21)

Reply:
Thank you for catching it!

Updated in https://github.com/woocommerce/woocommerce/pull/38405/commits/b731c4b3d0587890966afa84add8549526befb7b